Physics/particle.h

```diff
@@ -0,0 +1,60 @@
+#include "core.h"
+
+namespace AE86 {
+	/**
+	 * A particle is the simplest object that can be simulated in the
+	 * physics system. In reality, it is a point mass as per Newton's
+	 * Laws of motion.
+	 * Author: Jas Singh
+	*/
+	class Particle {

```

Comment:
I can see this confusing people down the road as a class name, but it should be fine.

Reply:
Renamed to PointMass XDD